src/lpython/utils.h

```diff
@@ -4,11 +4,26 @@
 #include <string>
 #include <libasr/utils.h>
 
+#include <sys/types.h>
+#include <sys/stat.h>
+#ifndef _WIN32
+    #include <unistd.h>
+#endif
+
+#ifdef _WIN32
+    #define stat _stat
+    #if !defined S_ISDIR
+        #define S_ISDIR(m) (((m) & _S_IFDIR) == _S_IFDIR)
+    #endif
+#endif
```

Comment:
This needs to be moved to the `.cpp` file and later we'll move it to a dedicated file, since we want most of our C++ files to be platform agnostic.

Reply:
Yup, I will move this to `utils.cpp` file.